experimental/packages/opentelemetry-sdk-metrics-base/src/export/PeriodicExportingMetricReader.ts

```diff
@@ -62,7 +63,16 @@ export class PeriodicExportingMetricReader extends MetricReader {
 
   private async _runOnce(): Promise<void> {
     const metrics = await this.collect({});
-    await this._exporter.export(metrics);
+    await this._exporter.export(metrics, result => {
+      if (result.code !== ExportResultCode.SUCCESS) {
+        globalErrorHandler(
+          result.error ??
+            new Error(
+              `PeriodicExportingMetricReader: metrics export failed (status ${result})`
```

Comment:
nit pick but since the ExportResultCode is a enum this will be print `status 1` or `status 0` which isn't practical i think

Reply:
it's clearly an error and status is always same which isn't really helpful, updated to include the `result.error` instead